src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -97,29 +101,29 @@ private function validate($content, $file = null)
         return array('file' => $file, 'valid' => true);
     }
 
-    private function display(InputInterface $input, OutputInterface $output, SymfonyStyle $io, $files)
+    private function display(SymfonyStyle $io, array $files)
     {
-        switch ($input->getOption('format')) {
+        switch ($this->format) {
             case 'txt':
-                return $this->displayTxt($output, $io, $files);
+                return $this->displayTxt($io, $files);
             case 'json':
                 return $this->displayJson($io, $files);
             default:
-                throw new \InvalidArgumentException(sprintf('The format "%s" is not supported.', $input->getOption('format')));
+                throw new \InvalidArgumentException(sprintf('The format "%s" is not supported.', $this->format));
         }
     }
 
-    private function displayTxt(OutputInterface $output, SymfonyStyle $io, $filesInfo)
+    private function displayTxt(SymfonyStyle $io, array $filesInfo)
     {
         $countFiles = count($filesInfo);
         $erroredFiles = 0;
 
         foreach ($filesInfo as $info) {
-            if ($info['valid'] && $output->isVerbose()) {
+            if ($info['valid'] && $this->displayCorrectFiles) {
```

Comment:
@javiereguiluz I was glad to use it, but it make tests failing as the verbosity methods are available since `3.1` only, dev requirements are to `~2.8|~3.0`. Should I up the version bound  to `~3.1` for that?